internal/backend/crypto/age/age.go

```diff
@@ -41,7 +42,7 @@ type Age struct {
 
 // New creates a new Age backend
 func New() (*Age, error) {
-	cDir, err := cache.NewOnDisk("age-github")
+	cDir, err := cache.NewOnDisk("age-github", 6*time.Hour)
```

Comment:
This default value should be documented in our doc, no?

Reply:
Eventually, yes. But for now age is really nothing more than a preview.
I don't want to spend too much time on writing documentation for something that might change completely.